cmd/policy-assistant/pkg/cli/analyze.go

```diff
@@ -296,7 +308,10 @@ func shouldIncludeANPandBANP(client *kubernetes.Clientset) (bool, bool) {
 	return includeANP, includeBANP
 }
 
-func VerdictWalkthrough(policies *matcher.Policy) {
+func VerdictWalkthrough(policies *matcher.Policy, sourceWorkloadTraffic string, destinationWorkloadTraffic string, port int, protocol string) {
```

Comment:
can we print a warning and exit the function for bad input?

I guess that would entail:
1. either traffic peer not found
2. `v1.Protocol(protocol)` not equal to TCP, UDP, SCTP